setup.py

```diff
@@ -86,9 +86,7 @@ def get_extras_require() -> Dict[str, List[str]]:
                 "dask[dataframe]",
                 "dask-ml",
                 "keras",
-                # TODO(toshihikoyanase): Remove the version constraint after resolving the issue
-                # https://github.com/optuna/optuna/issues/997.
-                "pytorch-lightning<0.7.0",
+                "pytorch-lightning>=0.7.1",
```

Comment:
We may not need to install `pytorch-lightning` for Python 3.5 since the supported version of PyTorch Lightning is 3.6 or newer. But, this is a trivial issue and I'll create a followup if necessary.

Reply:
I submitted a PR (https://github.com/optuna/optuna/pull/1193). If you have any comments, please let me know.